packages/quantic/force-app/main/default/lwc/quanticRecentResultsList/__tests__/quanticRecentResultList.test.js

```diff
@@ -0,0 +1,474 @@
+  () => ({default: 'Recent Results'}),
+  {
+    virtual: true,
+  }
+);
+jest.mock(
+  '@salesforce/label/c.quantic_Expand',
+  () => ({default: 'Expand Recent results'}),
+  {
+    virtual: true,
+  }
+);
+jest.mock(
+  '@salesforce/label/c.quantic_Collapse',
+  () => ({default: 'Collapse Recent results'}),
+  {
+    virtual: true,
+  }
+);
+
+const mockResults = [
+  {
+    title: 'Result 1',
+    clickUri: 'https://example.com/result1',
+    uri: 'https://example.com/result1',
+    excerpt: 'This is the first result.',
+  },
+  {
+    title: 'Result 2',
+    clickUri: 'https://example.com/result2',
+    uri: 'https://example.com/result2',
+    excerpt: 'This is the second result.',
+  },
+];
+
+const defaultRecentResultsTitle = 'Recent Results';
+const defaultResults = [];
+const defaultMaxLength = 10;
+let isInitialized = false;
+let updateRecentResultsState;
+
+const initialRecentResultsListState = {
+  results: defaultResults,
+  maxLength: defaultMaxLength,
+};
+let recentResultsListState = initialRecentResultsListState;
+
+const functionMocks = {
+  buildRecentResultsList: jest.fn(() => ({
+    state: recentResultsListState,
+    subscribe: functionMocks.recentResultsListSubscriber,
+    clear: functionMocks.clear,
+  })),
+  recentResultsListSubscriber: jest.fn((callback) => {
+    updateRecentResultsState = async () => {
+      callback();
+      await flushPromises();
+    };
+    return () => {};
+  }),
+  recentQueriesListUnsubscriber: jest.fn(),
+  clear: jest.fn(),
+};
+
+const defaultOptions = {
+  engineId: exampleEngine.id,
+  maxLength: defaultMaxLength,
+  label: 'Recent Results',
+  hideWhenEmpty: false,
+  target: '_self',
+  isCollapsed: false,
+};
+
+const selectors = {
+  initializationError: 'c-quantic-component-error',
+  placeholder: 'c-quantic-placeholder',
+  quanticCardContainer: 'c-quantic-card-container',
+  recentResultItem: '[data-testid="recent-result-item"]',
+  toggleButton: '[data-testid="action-button"]',
+  quanticRecentResultLink: 'c-quantic-recent-result-link',
+};
+
+const createTestComponent = buildCreateTestComponent(
+  QuanticRecentResultsList,
+  'c-quantic-recent-results-list',
+  defaultOptions
+);
+
+function prepareHeadlessState() {
+  // @ts-ignore
+  mockHeadlessLoader.getHeadlessBundle = () => {
+    return {
+      buildRecentResultsList: functionMocks.buildRecentResultsList,
+    };
+  };
+}
+
+function mockSuccessfulHeadlessInitialization() {
+  // @ts-ignore
+  mockHeadlessLoader.initializeWithHeadless = (element, _, initialize) => {
+    if (element instanceof QuanticRecentResultsList && !isInitialized) {
+      isInitialized = true;
+      initialize(exampleEngine);
+    }
+  };
+}
+
+function mockErroneousHeadlessInitialization() {
+  // @ts-ignore
+  mockHeadlessLoader.initializeWithHeadless = (element) => {
+    if (element instanceof QuanticRecentResultsList) {
+      element.setInitializationError();
+    }
+  };
+}
+
+describe('c-quantic-recent-queries-list', () => {
```

Comment:
```suggestion
describe('c-quantic-recent-results-list', () => {
```

Reply:
oops x) really wanted to go to vacation with nothing in my head haha